edit portfolio.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="stylesheet.css" />
+    <header>
+      <div class="login">
+        <ul>
+          <li><a href="portfolio.html">Portfolio</a></li>
+          <li><a href="signup.html">login</a></li>
+        </ul>
+      </div>
+    </header>
+  </head>
+  <body>
+    <div class="main-container">
+      <div class="personal-info">
+        <div class="about-sec">
+          <h2>About</h2>
+          <p>
+            its Mishal firdous here, i am a software engineer, Previsiosly
+            having one year CMS expertices in wordpress, wix and shopify.Which
+            plays a critical role in the technology industry. Through these
+            content management system design would more eye cautching although
+            responsible for designing, creating, and maintaining software that
+            meets the needs of businesses and users in various domains,
+            including finance, healthcare, e-commerce, entertainment, and many
+            others. Software engineers work with programming languages,
+            algorithms, and software development methodologies to build
+            applications and systems that drive modern society.
+          </p>
+        </div>
+        <div class="experince-sec">
+          <div class="container">
+            <h2>Experince</h2>
+            <button id="addButton">Add</button>
+            <div class="experiences">
+              <div id="experienceList"></div>
+            </div>
+          </div>
+
+          <div
+            id="experienceForm"
+            class="form"
+            action="index.html"
+            method="POST"
+          >
+            <h2>Add Experience</h2>
+            <label for="companyName"></label>
+            <input
+              type="text"
+              id="companyName"
+              placeholder="Company Name:"
+              required
+            />
+            <label for="startDate"></label>
+            <input type="date" id="startDate" required />
+            <label for="endDate"></label>
+            <input type="date" id="endDate" required />
+            <label for="description"></label>
+            <textarea
+              id="description"
+              rows="4"
+              placeholder="Description"
+              required
+            ></textarea>
+            <button id="saveButton">Save</button>
+          </div>
```

Comment:
judging by the attributes and classnames, this appears to be a form thats submitted on clicking save.
this form should start with `<form>` tag. define the button type as submit. redo this whole form according to the best practices. You can read more about form [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form)

Reply:
fixed!